### ISSUE TITLE
doc/words/man/im.1 remains, but im was removed

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5025]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5025|
|Type|defect|
|Created|2011-04-04 21:20:22Z|
|Branch|https://github.com/twisted/twisted/tree/im-man-5025|

At various points ([#3507](https://github.com/twisted/twisted/issues/3507), [#3340](https://github.com/twisted/twisted/issues/3340), [#3699](https://github.com/twisted/twisted/issues/3699)) we got rid of the actual im client from Twisted Words.  But the man page for im remains.

<details><summary>Searchable metadata</summary>

```
trac-id__5025 5025
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_im_man_5025 branches/im-man-5025
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__words words
keywords__ 
time__1301952022000000 1301952022000000
changetime__1306865557000000 1306865557000000
version__None None
owner__thijs thijs
cc__ralphm cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [31987]) Branching to 'im-man-5025'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [31989]) remove obsolete man page for im client, refs [#5025](https://github.com/twisted/twisted/issues/5025)

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set owner to @thijstriemstra|
|-|-|

Cool, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [32005]) Merge im-man-5025: Removed man page for im client.

Author: thijs
Reviewer: therve
Fixes: [#5025](https://github.com/twisted/twisted/issues/5025)
